lib/versioning/composer/index.spec.ts

```diff
@@ -148,6 +156,36 @@ describe('semver.getNewValue()', () => {
       })
     ).toEqual('^1.0');
   });
+  it('bumps caret to same', () => {
+    expect(
+      semver.getNewValue({
+        currentValue: '^1.0.0',
+        rangeStrategy: 'bump',
+        fromVersion: '1.0.0',
+        toVersion: '1.3.5',
+      })
+    ).toEqual('^1.0');
```

Comment:
This is not correct. `^1.3.5` is correct for `bump`

Reply:
I missed this in copy-paste. :)